changelogs/fragments/jira_improvements.yaml

```diff
@@ -0,0 +1,6 @@
+bugfixes:
+  - Fixed examples documentation ().
+  - Module no longer incorrectly reports change for information gathering operations ().
+  - Replaced custom parameter validation with ``required_if`` ().
+minor_changes:
+  - Added the traceback output to ``fail_json()`` calls deriving from exceptions ().
```

Comment:
This file needs an update :)

Reply:
Yes, as there was no issue # I was waiting for the PR #, and then I started another bugfix and forgot that. Just pushed an update.